lib/create.js

```diff
@@ -53,23 +53,39 @@ var toUnderscoreFromCamel = function(string) {
 };
 
 var Create = {
-  contract: function(directory, name, callback) {
+  contract: function(directory, name, options, callback) {
+    if(typeof options == "function") {
+      callback = options;
+    }
+
     var from = templates.contract.filename;
     var to = path.join(directory, name + ".sol");
 
+    if (!options.force && fs.existsSync(to)) {
+      console.error('Can not create ' + name + '.sol: file exists');
```

Comment:
Just a note: normal convention is to `callback(err)` and let "upstream processing" handle the outputting. At the very least, `config.logger` should be used in place of `console`. But I think in this case `callback(err)` makes more sense.

Reply:
Updated!